docs/Gemfile.lock

```diff
@@ -234,8 +234,7 @@ GEM
     minitest (5.25.5)
     net-http (0.6.0)
       uri
-    nokogiri (1.18.9)
-      mini_portile2 (~> 2.8.2)
+    nokogiri (1.18.9-x86_64-linux-gnu)
```

Comment:
hmm. not sure how this leaked in.  can you revert the lock file?

Reply:
reverted the whole lock file